pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -86,6 +86,8 @@ type ServiceBindingRequestStatus struct {
 	BindingStatus BindingStatus `json:"BindingStatus"`
 	// Secret is the name of the intermediate secret
 	Secret string `json:"Secret"`
+	// LabelObjects contains all the objects with the label
+	LabelObjects []string `json:"LabelObjects"`
```

Comment:
How about naming it as `ApplicationObjects`?

Reply:
@baijum hmm.. that's a better word..

Reply:
done here 
5fe2f13